### PR TITLE
Showcase

### PR DESCRIPTION
**Pull request by philecs**

### CODE REVIEW

File: docs/oihPostmanGuide.md [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+      - [Add a new User](#add-a-new-user)
+      - [Delete a User](#delete-a-user)
+      - [Update a User](#update-a-user)
+  - [Examples](#examples)
+    - [Flow Examples](#flow-examples)
+    - [Component Examples](#component-examples)
+
+## Prerequisites
+
+In order to use the Open Integration Hub we provided a demo environment and a [postman collection](postman/OIH_Framework_Showcase.postman_collection.json) and a [postman environment](OIH_Framework.postman_environment.json) to test the functionalities of the Open Integration Hub.
```
Comment (fdegenhr) [inline docs/oihPostmanGuide.md]: postman environment link not working, use postman/OIH_Framework.postman_environment.json

File: docs/oihPostmanGuide.md [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+      - [Get all Users](#get-all-users)
+      - [Add a new User](#add-a-new-user)
+      - [Delete a User](#delete-a-user)
+      - [Update a User](#update-a-user)
+  - [Examples](#examples)
+    - [Flow Examples](#flow-examples)
+    - [Component Examples](#component-examples)
+
+## Prerequisites
+
```
Comment (fdegenhr) [inline docs/oihPostmanGuide.md:34]: Add short paragraph about downloading postman

File: docs/oihPostmanGuide.md [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+It is necessary to add valid user data in order to perform the requests.
+Please process the following steps in order to be able to generate a valid token for the requests:
+
+1. Right click on the `OIH_SHOWCASE_COLLECTION` and click on `edit` OR click on the three dots `...` and click `edit`.
+2. Jump to `Pre-request Scripts`
+3. Replace `{YOUR_USERNAME}` with the username you received
+4. Replace `{YOUR_PASSWORD}` with the password you received
+5. Click on `Update`
+
```
Comment (fdegenhr) [inline docs/oihPostmanGuide.md:53]: Is this easier?
const username = 'franz.degenhardt@x-integrate.com';
const password = 'xyz123;
const echoPostRequest = {
  url: `http://iam.openintegrationhub.com/login?username=${username}&password=${password}`,
  method: 'POST',
  header: 'Content-Type:application/json'
};

**Review by fdegenhr — APPROVED** [review]

File: docs/oihPostmanGuide.md (hunk 3 above)
Comment (philecs) [reply docs/oihPostmanGuide.md:53]: Good point. I'll update this.

File: docs/oihPostmanGuide.md (hunk 2 above)
Comment (philecs) [reply docs/oihPostmanGuide.md:34]: done

File: docs/oihPostmanGuide.md (hunk 3 above)
Comment (philecs) [reply docs/oihPostmanGuide.md:53]: Done

File: docs/oihPostmanGuide.md (hunk 1 above)
Comment (philecs) [reply docs/oihPostmanGuide.md]: Done

**Pull request merged**